src/router/client.js

```diff
@@ -80,6 +84,26 @@ const router = new VueRouter({
   },
 });
 
+// eslint-disable-next-line
+router.beforeEach((to, from, next) => {
```

Comment:
the redirect should happen on the server, not the on client.

Reply:
@brenoferreira first I changed in `router/server.js` but its not working in there so, I then changed it in client-side, Should I need to change the 'serverEntry` to get it worked?

Reply:
Yes. Server is accessible via port 8080

Reply:
Yess, I added it to `server`